draft-ietf-quic-transport.md

```diff
@@ -3557,12 +3550,15 @@ which could prevent the connection from ever becoming idle.  Non-ack-eliciting
 packets are eventually acknowledged when the endpoint sends an ACK frame in
 response to other events.
 
-In order to assist loss detection at the sender, an endpoint SHOULD send an ACK
-frame immediately on receiving an ack-eliciting packet that is out of order. The
-endpoint SHOULD NOT continue sending ACK frames immediately unless more
-ack-eliciting packets are received out of order.  If every subsequent
-ack-eliciting packet arrives out of order, then an ACK frame SHOULD be sent
-immediately for every received ack-eliciting packet.
+In order to assist loss detection at the sender, an endpoint SHOULD generate
+and send an ACK frame without delay when it receives an ack-eliciting packet
+either:
+
+* when the received packet has a packet number less than another ack-eliciting
+  packet that has been received, or
+* when the packet has a packet number larger than the highest-numbered
+  ack-eliciting packet that has been received and there are missing packets
```

Comment:
This wording strikes me as odd.  A receiver does not know whether there are _missing packets_.  We should simply state that there is a gap in the packet numbers.

Reply:
@dtikhonov : _Missing packets_ is not the same as _lost packets_. The receiver knows when packets are missing, it just doesn't know whether they are reordered or lost.